addons/website/static/src/services/website_custom_menus.js

```diff
@@ -53,7 +53,8 @@ registry.category('website_custom_menus').add('website.menu_edit_menu', {
     Component: EditMenuDialog,
     isDisplayed: (env) => !!env.services.website.currentWebsite
         && env.services.website.isDesigner
-        && !env.services.ui.isSmall,
+        && !env.services.ui.isSmall
+        && !env.services.website.currentWebsite.metadata.translatable,
```

Comment:
Commit message is confusing, it says that the edit menu dialog is displayed in the user's language instead of the displayed website's language... but the only fix here is about hiding it when not displaying the website in its default language (as we chose). But does that mean that the edit menu dialog is still displayed in the user's language rather than website language ? :thinking: I tried to test... but apparently changing backend language does not work anymore -_-

Also, with this commit, the edit menu is properly hidden in translated websites... but not in translate mode :/ Was it like that before? It's even more important to fix it there as after you use, it reloads the iframe and breaks the translate mode.

Reply:
I updated the commit to always display the menus from the "Edit Menu" in the website's language, and hide that menu for a translatable website.
For the popover shown in translate mode, it was a bug coming from https://github.com/odoo/odoo/commit/df1869153a90898bad3e0b61e5fc43a9ed3d59c9